src/components/Button.js

```diff
@@ -1,16 +1,29 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import styles from './Button.scss';
+import styles from '../components/Button.scss';
 
-const Button = ({ onClick, children, tabIndex, ...props }) =>
-  (<button className={styles.button} tabIndex={tabIndex} {...props} onClick={() => onClick()}>
+const buttonType = {
+  square_button: styles.square_button,
+  round_button: styles.roundblue_button,
+  round_button_header: styles.roundblue_button_header,
+  border_button: styles.blueborder_button,
+};
```

Comment:
If you do the inlining thing I suggest below, you will not need this lookup table anymore.